cypress/e2e/publicPaths/render/recoverPassword/index.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { runTestsForDevices } from "../../../utils";
+import { allDevicesForAppLayout } from "../../../utils/devices";
+import { newExpectation, newShouldArgs } from "../../../utils/helpers";
+
+const pageName = "Reset password page";
+const fakeToken = "this-is-a-fake-token";
+const currentPage = `/recover-password?token=${fakeToken}`;
+
+describe(`${pageName} renders expected components on different devices`, () => {
+  const tests = [
+    newExpectation(
+      "should have reset password component with correct title",
+      "[data-cy=reset-password-page]",
+      newShouldArgs("be.visible.and.contain", [
+        "Reset your password",
+        "Please enter a new password below.",
+      ]),
+    ),
+    newExpectation(
+      "should have reset password form with inputs",
+      "[data-cy=reset-password-form] input",
+      newShouldArgs("be.visible.and.have.length", 2),
```

Comment:
We define some `newShouldArgs` here https://github.com/dolthub/hosted-cypress/blob/main/cypress/e2e/utils/sharedTests/sharedFunctionsAndVariables.ts#L4

Would it be worth adding some `beVisiableAndHaveLength` in that file?